packages/admin/admin/src/mui/menu/CollapsibleItem.tsx

```diff
@@ -185,8 +185,8 @@ export const MenuCollapsibleItem = (inProps: MenuCollapsibleItemProps) => {
                     sx={{
                         pointerEvents: "none",
                     }}
-                    open={isSubmenuOpen}
-                    anchorEl={anchorEl}
+                    open={isSubmenuOpen && anchorEl !== null}
+                    anchorEl={anchorEl ?? null}
```

Comment:
```suggestion
                    anchorEl={anchorEl}
```

This is not necessary since the default value is `null`.